ci/run.sh

```diff
@@ -1,6 +1,14 @@
+#!/bin/bash
+current_dir="$(dirname "$0")"
+
 bundle install
 bundle exec rake gradle.properties
 ./gradlew assemble
 bundle exec rake vendor
+
+# we don't want to bundle ASN DB with this gem, so we download this just for tests.
+curl -sL http://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz > GeoLite2-ASN.tar.gz
```

Comment:
@suyograo you could handle the downloading and extracting via Gradle like I did for the user agent parser https://github.com/logstash-plugins/logstash-filter-useragent/blob/master/build.gradle#L65

Would be a little more portable and less hacky than `dirname` + `curl`.